README.md

```diff
@@ -161,14 +188,27 @@ class Config : public Owned<::z_owned_config_t> {
 }
 ```
 
-Classes which protects corresponding [zenoh-c](https://github.com/eclipse-zenoh/zenoh-c) so-called `owned` structures from copying, allowing move semantic only. Corresponding utility functions like `z_check`, `z_null`, `z_drop` are integrated into the `Owned` base template.
+Classes which protects corresponding so-called `owned` structures from copying, allowing move semantic only. Corresponding utility functions like `z_check`, `z_null`, `z_drop` are integrated into the `Owned` base template.
 
 * Closures
 
-It's classes representing [zenoh-c](https://github.com/eclipse-zenoh/zenoh-c) callback structures:
+It's classes representing callback structures:
 ```C++
 typedef ClosureMoveParam<::z_owned_closure_reply_t, ::z_owned_reply_t, Reply> ClosureReply;
 typedef ClosureConstPtrParam<::z_owned_closure_query_t, ::z_query_t, Query> ClosureQuery;
 ```
 
 They allows to wrap C++ invocable objects (fuctions, lambdas, classes with operator() overloaded) and pass them as callbacks to zenoh.
+
+
+[zenoh-c]: https://github.com/eclipse-zenoh/zenoh-c
+[zenoh-cpp]: https://github.com/eclipse-zenoh/zenoh-cpp
+[zenohpico]: https://github.com/eclipse-zenoh/zenoh-pico
+[zenoh.hxx]: https://github.com/eclipse-zenoh/zenoh-cpp/blob/main/include/zenoh.hxx 
+[zenohc.hxx]: https://github.com/eclipse-zenoh/zenoh-cpp/blob/main/include/zenohc.hxx 
+[zenohpico.hxx]: https://github.com/eclipse-zenoh/zenoh-cpp/blob/main/include/zenohpico.hxx 
```

Comment:
Stylistic question. Do we want to use `zenohpico` or `zenoh-pico`? I lean more towards the second option.

Reply:
I think we shoud use zenoh-c and zenoh-pico as library names in documentation, but zenohc and zenohpico in identifiers